Create 02-11-22_writeup.md (#183)
very-good-science/data-ethics-club: site/write_ups/2022/02-11-22_writeup.md

Research from 1970s has shown black males having disproportionate restraints/drug doses in mental health care. Why has nothing been done about this? Stories stick around longer even when they are disproved with research - the example of vitamin D was mentioned.

Many of us were aware of the research bias. For example; how psychological research studies are done on psychology students or how medical text books show e.g. skin conditions on white people or autism diagnoses. Even within biology (for example reproductive biology), people thought of how important sperm was until female scientists came along, then they thought about the eggs

>  https://web.stanford.edu/~eckert/PDF/Martin1991.pdf  - The Egg and the Sperm: How Science Has Constructed a Romance Based on Stereotypical Male-Female Roles. Article - https://bigthink.com/surprising-science/there-is-inherent-sexism-in-how-we-talk-about-biology/

Reviewer: ```suggestion
e.g.  [The Egg and the Sperm: How Science Has Constructed a Romance Based on Stereotypical Male-Female Roles.](https://web.stanford.edu/~eckert/PDF/Martin1991.pdf) or [this article](https://bigthink.com/surprising-science/there-is-inherent-sexism-in-how-we-talk-about-biology/).
```

> 60% of people who died from covid were disabled - this was disproportionate even when those disabilities were nothing to do with respitory diseases e.g. lung function/immune system. This is partially because anyone with a learning disabilities had a do not rescucitate order as a default. Some of these people were on the one hand "essential workers" and on the other hand disposable. And there was a similar talking point to the one in the article, "genetics must be causing this issue", in the media with the "underlying health conditions" framing.

Reviewer: ```suggestion
60% of people who died from covid were disabled - this was disproportionate even when those disabilities were nothing to do with respiratory diseases e.g. lung function/immune system. This is partially because anyone with a learning disability had a do not resuscitate order as a default. Some of these people were on the one hand "essential workers" and on the other hand disposable. And there was a similar talking point to the one in the article, "genetics must be causing this issue", in the media with the "underlying health conditions" framing.
```

How people are racialised by society is distinct from their identity. Most ethnicity data that is self-reported is probably based on their own experience of how society has racialised them throughout their lives. But NHS ethnicity data is not self-reported at the moment. And it's very difficult to change once someone else has recorded for you. Should UK also record class?

Census has evolved over time, but still requires people to sort themselves into boxes defined by someone else. We should make sure a self-reported ethnicity option is available alongside the prescribed boxes! In healthcare the second most popular category is "not known" (after "white british"). Need to break down ethnicity to include things like background, individual history etc, and not just base it off a superficial judgement of skin colour. Scapegoating of minorities is such a fundamental part of British politics/media that it's difficult to tackle these issues properly.

## How do we actually  measure and record factors such as race?

> "How do I identify" is a pretty good measure. How do you see yourself? It depends on what you want. Do you want a 23andme type of thing which is genetic based? Or is it based on appearance on social? Should we establish what are the risk factors for illnesses and then just check for those? If I'm checking for covid, maybe I just need to know "are all your friends vaccinated?". Whilst that may be affected by social factors, why not go straight to the source. What do I mean by this when I ask by race? What do I want when I want this information? In [UK Census](https://www.southampton.ac.uk/ceip/publications/ethnic-identity-in-the-census.page#:~:text=In%20the%20English%20Census%20now,to%20be%20Black%20and%20English) there was a weird thing where you could be White English but only Black British.

Reviewer: ```suggestion
"How do I identify" is a pretty good measure. How do you see yourself? It depends on what you want. Do you want a 23andme type of thing which is genetic based? Or is it based on appearance on social factors? Should we establish what are the risk factors for illnesses and then just check for those? If I'm checking for covid, maybe I just need to know "are all your friends vaccinated?". Whilst that may be affected by social factors, why not go straight to the source. What do I mean by this when I ask by race? What do I want when I want this information? In [UK Census](https://www.southampton.ac.uk/ceip/publications/ethnic-identity-in-the-census.page#:~:text=In%20the%20English%20Census%20now,to%20be%20Black%20and%20English) there was a weird thing where you could be White English but only Black British.
```

## Case Study: Data Collection Practises in the EU (in particular, Sweden)

In Sweden, no ethnicity grouping is allowed to be recorded. The [intention](https://www.thelocal.se/20180508/why-sweden-doesnt-keep-stats-on-ethnic-background-and-crime/) is to prevent biases being built into the system. 

In a more comprehensive [Analysis and comparative review of equality data collection practises in the European Union](https://ec.europa.eu/info/sites/default/files/data_collection_in_the_field_of_ethnicity.pdf), this is noted in context amongst other country:

"Sweden: Race is not used as the term is considered to be closely linked to racism.
Ethnic origin (etniskt ursprung) is more accepted, used predominantly within the
anti-discrimination area. The politically correct term used in public documents is
foreign origin as opposed to Swedish origin. These two terms are officially defined

> by Statistics Sweden as follows.31 Persons of Swedish origin are persons born in

Reviewer: ```suggestion
by Statistics Sweden as follows: Persons of Swedish origin are persons born in
```

## Saini also refers to the overlapping of different inequalities and privileges in different contexts. How does this intersectionality of identities impact the way we do or should analyse data about demographics?

Intersectionality is important, but the more intersectional a group you want to look at, the more difficult the data is to get (especially in big enough numbers to apply machine learning to).

We touched on this earlier, but splitting categories seems sensible (although numbers get small in increasingly narrow categories). Ethnicity recording across the National Health Service (NHS) has improved dramatically over the past decade. [This study](https://pubmed.ncbi.nlm.nih.gov/24323951/) profiles the completeness, consistency and representativeness of routinely collected ethnicity data in both primary care and hospital settings. The second greatest category in ethnicity recorded is second after White British. The question is if this group is still predominantely White British or not.

> 

Reviewer: ```suggestion

As we mentioned earlier, perhaps the most useful thing to do is just to ask people what you need to know for each context at a time. If we need to know if someone is privileged or disadvantaged in a particular context regarding a particular characteristic, we should probably just ask about that instead of trying to find a catch-all measure that will never work all of the time. 
That way we can gather information that is relevant to what we need to know, and not rely on proxy values.
After all, humans are not one-dimensional, and our identities aren't either!
```
